htlcswitch/link.go

```diff
@@ -2195,6 +2165,36 @@ func (l *channelLink) HtlcSatifiesPolicy(payHash [32]byte,
 		return &lnwire.FailExpiryTooFar{}
 	}
 
+	// Next, using the amount of the incoming HTLC, we'll calculate the
```

Comment:
commit message should explain the rationale behind this change.

Reply:
added